Monster.cpp

```diff
@@ -4,14 +4,14 @@
 
 std::ostream &operator<<(std::ostream &os, const Monster &ch)

```

Comment:
Jelenleg szerintem ezt kivehetitek, mert a jelenlegi featurehoz nem szükséges, viszont ebből nekünk is volt gond, hogy ha ez bent van bármikor meghívható és lehet hiba.

Reply:
Ezt inkább bent hagynánk, mert gyors és könnyű kiírási lehetőséget biztosít a karakterek számára.